test/factories/puppet_related.rb

```diff
@@ -71,6 +71,14 @@
     end
   end
 
+  factory :puppet_lookup_value do
+    sequence(:value) {|n| "value#{n}" }
+
+    trait :with_use_puppet_default do
```

Comment:
Shouldn't the old `:with_use_puppet_default` trait be deleted/moved from the lookup_value factory?